index.js

```diff
@@ -793,6 +611,40 @@ app.post(
   },
 )
 
+app.post('/api/startChatbot', Validator.body(['coreid', 'event']).exists(), async (request, response) => {
+  try {
+    const validationErrors = Validator.validationResult(request).formatWith(helpers.formatExpressValidationErrors)
+
+    if (validationErrors.isEmpty()) {
+      const coreId = request.body.coreid
+      const alertType = request.body.event.split(' ')[0]
```

Comment:
This event-split-thing seems pretty ugly. Why was it decided that this would be how the firmware and webhook would pass the data to this endpoint?

Reply:
I don't think there's any reason in particular. The firmware could just as easily send just "Duration" or "Stillness" - but that's a change across the other repo as well.